sound/soc/sof/sof-client-probes.c

```diff
@@ -520,7 +520,7 @@ static ssize_t probe_points_read(struct file *file, char __user *to,
 
 	ret = pm_runtime_get_sync(dev);
 	if (ret < 0 && ret != -EACCES) {
-		dev_err_ratelimited(dev, "error: debugfs read failed to resume %d\n", ret);
+		dev_err_ratelimited(dev, "debugfs read failed to resume %d\n", ret);
```

Comment:
all these cleanups could be done in the initial code, before moving to the client-based solution, no?

Reply:
I rather not backport these, we agreed that we will not do `s/error: /` for existing code and the rest is really just naming consistency among the client drivers.